Ui/Component/Listing/Column/SyncedOrder.php

```diff
@@ -57,9 +68,14 @@ public function prepareDataSource(array $dataSource)
     {
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as & $item) {
-                $order = $this->orderRepository->get($item['entity_id']);
                 $orderSyncDate = '';
 
+                try {
+                    $order = $this->orderRepository->get($item['entity_id']);
+                } catch (NoSuchEntityException $e) {
+                    $this->logger->log($e->getMessage() . ', entity id: ' . $item['entity_id']);
+                }
+
                 if ($order->getTjSalestaxSyncDate()) {
```

Comment:
If the exception is caught, `$order` may not be set. Should we check to see if `$order` exists for this conditional to avoid an additional error?

```php
if ($order && $order->getTjSalestaxSyncDate())
```